cmake/pythonFormat.cmake

```
@@ -0,0 +1,34 @@
+# Additional target to run python linters and formatters on python scripts
+#
+# Requires black/flake8 to be available in the environment
+
+
+# Get all our Python files
+file(GLOB_RECURSE ALL_PYTHON_FILES ${PROJECT_SOURCE_DIR}/*.py)
```

```suggestion
file(GLOB_RECURSE ALL_PYTHON_FILES ${PROJECT_SOURCE_DIR}/*.py)
```
This will also pick up a potential `install` folder from a (previous) build.

---

Yes, now restricted to our own Python source directories, so it's a but cleaner